tree-sitter-java/TreeSitter/Java/AST.hs

```diff
@@ -1,4 +1,13 @@
-{-# LANGUAGE DataKinds, DeriveAnyClass, DeriveGeneric, DeriveTraversable, DuplicateRecordFields, TemplateHaskell, TypeApplications #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
I really love that you’re standardizing this 👍 Would love to hear what motivates your selection of this style when you have the time, obvs. not urgent tho.

Reply:
it's easier to edit (add/remove), vertical lists are more readable to me than infinite left scroll.

Reply:
Rad, thanks! (Maybe we should adopt this in `semantic` &c. too.)